Makefile

```diff
@@ -23,9 +23,11 @@ install-hooks: venv
 		venv/bin/pre-commit install -f --install-hooks
 
 venv: vendor/venv-update requirements.txt requirements-dev.txt
-		vendor/venv-update \
-				venv= -ppython3.7 venv \
-				install= -r requirements.txt -r requirements-dev.txt
+		# Install requirements.txt and requirements-dev.txt independently so that
+		# they can overlap on dependencies and not get duplicate requirement errors
+		# from pip
+		vendor/venv-update venv= -ppython3.7 venv install= -r requirements.txt
+		vendor/venv-update venv= -ppython3.7 venv install= -r requirements-dev.txt
```

Comment:
Ok I think this needs some more thought from me, since it's now causing issues where things in `requirements.txt` are pruned when running this with `make dev`, so that's pretty bad.

Reply:
Never mind, rebasing onto master fixed this, probably because #80 had some requirement changes that fixed this.